app/controllers/admin/categories_controller.rb

```diff
@@ -59,13 +59,16 @@ def category_params
     :front_page,
     :active,
     :section,
-    :rank
+    :rank,
+    :team_list,
```

Comment:
Is this actually used?  I remember we discussed scoping by team, but I didn't see it, or missed it...